pytest.ini

```diff
@@ -6,5 +6,7 @@ flake8-ignore =
     setuptools/py*compat.py F811
 doctest_optionflags=ELLIPSIS ALLOW_UNICODE
 filterwarnings =
+	# Fail on warnings
+	error
 	# https://github.com/pypa/setuptools/issues/1823
 	ignore:bdist_wininst command is deprecated
```

Comment:
The file also uses spaces, should these tabs be spaces too?

Reply:
Yes, indentation should be consistent.